### PR TITLE
Updating Connect-MsGraph line

### PR DESCRIPTION
**Pull request by get-itips**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/5772

User isn't a parameter of Connect-MsGraph

### CODE REVIEW

**Comment by get-itips** [comment]
@mypil Joyce, please reference the needed people to make this merged. Thanks

File: windows/deployment/windows-autopilot/existing-devices.md [hunk 1]
```diff
@@ -70,13 +70,12 @@ See the following examples.
     Install-Module WindowsAutopilotIntune -Force
     ```
 
-3. Enter the following lines and provide Intune administrative credentials
-   - In the following command, replace the example user principal name for Azure authentication (admin@M365x373186.onmicrosoft.com) with your user account. Be sure that the user account you specify has sufficient administrative rights.
+3. Enter the following line and provide Intune administrative credentials
```
Comment (illfated) [inline windows/deployment/windows-autopilot/existing-devices.md]: I suggest adding a colon at the end of the sentence, to highlight what comes next:
```suggestion
3. Enter the following line and provide Intune administrative credentials:
```

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by Dansimp** [comment]
@greg-lindsay please review

**Pull request merged**